### PR TITLE
e-h-bus: Add MSRV notice and prepare 0.1.0-alpha.0 release

### PR DESCRIPTION
**Pull request by eldruin**
:tada:

### CODE REVIEW

File: .github/workflows/ci-bus.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches: [ staging, trying, master ]
+  pull_request:
+
+name: CI Async
```
Comment (Dirbaio) [inline .github/workflows/ci-bus.yml:6]:
```suggestion
name: CI bus
```

File: .github/workflows/ci-bus.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+env:
+  RUSTFLAGS: '--deny warnings'
+
+jobs:
+  ci-linux-async:
```
Comment (Dirbaio) [inline .github/workflows/ci-bus.yml:12]:
```suggestion
  ci-linux-bus:
```

File: .github/workflows/clippy-bus.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches: [ staging, trying, master ]
+  pull_request:
+
+name: Clippy check
+jobs:
+  clippy_check-async:
```
Comment (Dirbaio) [inline .github/workflows/clippy-bus.yml:8]:
```suggestion
  clippy_check-bus:
```

File: .github/workflows/ci-bus.yml (hunk 1 above)
Comment (eldruin) [reply .github/workflows/ci-bus.yml]: Thanks! I fixed it now (also below)

**Review by Dirbaio — APPROVED** [review]
:rocket:!!
 
bors r+

**Pull request merged**